packages/addons-v5/index.js

```diff
@@ -9,7 +9,7 @@ exports.topic = {
 
   Add-ons are created with the \`addons:create\` command, providing a reference
   to an add-on service (such as \`heroku-postgresql\`) or a service and plan
-  (such as \`heroku-postgresql:hobby-dev\`).
+  (such as \`heroku-postgresql:heroku-postgresql:standard-0\`).
```

Comment:
Looks like there's an extra `heroku-postgresql` here. Also, do we want to use `hobby-basic` instead of `standard-0` for the new example plan since that's the lowest cost paid plan?

Reply:
Good catch! Will update in similar places as well!